src/app/pages/common/ix/ix-form/ix-form.component.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  Component, EventEmitter, Input, Output,
+} from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { UntilDestroy } from '@ngneat/until-destroy';
+
+@UntilDestroy()
+@Component({
+  selector: 'ix-form',
```

Comment:
As per discussion IxForm and IxFieldset may create the same issues we had with old form implementation where if I want to add something else (like a third button) I may need to start adding more inputs to the component.
Let's see if we can use a different approach by extracting common styling to a css class / directive.